locales/nb_NO/pcsx2_Main.po

```diff
@@ -97,59 +97,51 @@ msgstr "Venter på oppgaven..."
 #: common/src/Utilities/wxAppWithHelpers.cpp:35
 msgid "Includes idle event processing and some other uncommon event usages."
 msgstr ""
-"Inkluderer hvilende prosessbehandlinger og noen andre uvanlige "
-"prosessbruksområder."
+"Inkluderer hvilende prosessbehandlinger og noen andre uvanlige prosessbruksområder."
 
 #: pcsx2/CDVD/InputIsoFile.cpp:248
 msgid "Unrecognized ISO image file format"
 msgstr "Ukjent ISO-filformat"
 
 #: pcsx2/Elfheader.cpp:163
 msgid "Cannot load ELF binary image.  The file may be corrupt or incomplete."
-msgstr ""
-"Kan ikke laste inn ELF-binærfilen. Filen kan være korrumpert eller "
-"ufullstendig."
+msgstr "Kan ikke laste inn ELF-binærfilen. Filen kan være korrumpert eller ufullstendig."
 
 #: pcsx2/Elfheader.cpp:165
 msgid ""
-"If loading from an ISO image, this error may be caused by an unsupported ISO "
-"image type or a bug in PCSX2 ISO image support."
+"If loading from an ISO image, this error may be caused by an unsupported ISO image type "
```

Comment:
Ditto. There are others as well, I believe PoEdit changed them automatically which is unnecessary.